hazelcast/src/main/java/com/hazelcast/client/impl/protocol/parameters/ExceptionResultParameters.java

```diff
@@ -33,24 +34,23 @@
      * ClientMessageType of this message
      */
     public static final int TYPE = ResponseMessageConst.EXCEPTION;
-    public String className;
-    public String causeClassName;
+    public int errorCode;
     public String message;
-    public String stacktrace;
+    public StackTraceElement[] stackTrace;
```

Comment:
Do we really need to communicate the stack trace for well known errors, e.g. AuthenticationFailure ? The stack trace exposes server internal operation to a client developer, should he/she be knowledgable to understand the stack trace? There may be a lot of questions from the client developers regarding this field .

what is a typical size for this field?